source.go

```diff
@@ -46,13 +46,14 @@ func (s *Source) Configure(ctx context.Context, cfg map[string]string) error {
 }
 
 func (s *Source) Open(ctx context.Context, pos sdk.Position) error {
+	sdk.Logger(ctx).Info().Bytes("position", pos).Msg("Opening a source...")
```

Comment:
We expect the position to be a JSON string, right? It would be more valuable to display it as a string then.
```suggestion
	sdk.Logger(ctx).Info().Str("position", string(pos)).Msg("Opening a source...")
```

Reply:
> We expect the position to be a JSON string, right?

At this point, yes.

Reply:
I did pay attention to this, so I checked the logs and saw that it was actually already "stringified". Then I checked the docs, and they actually confirm its adding the bytes as a string.

Reply:
TIL!